core/LSU.cpp

```diff
@@ -188,6 +219,11 @@ namespace olympia
             // however, we can ONLY update instruction status as SCHEDULED for a new issue event
 
             ILOG("Another issue event scheduled " << inst_ptr);
+
+            if(isReadyToIssueInsts_())
```

Comment:
should it be an assert instead of if(....), since we just updated issue priority? Does that mean we are guaranteed to find one inst that is ready?

Reply:
Yes isReadytoIssueInsts_ finds a instruction that can be issued.Yes if it returns true it should issue.